lib/cli/commands-schema/service.js

```diff
@@ -6,27 +6,6 @@ const noServiceCommands = require('./no-service');
 
 const commands = (module.exports = new Map());
 
-commands.set('package', {
```

Comment:
We should not move it to `aws-service` only, as that command is also supported for different providers.

Reply:
ok thanks, I looked at the deploy command and only found it in `aws-service.js` is this a aws only command?

Reply:
oh, that's a good point, I wasn't aware that's the case here. @medikoo Should `package` also be specific to AWS only here and we should require other providers to provide schema for that command explicitly?